### PR TITLE
feat(core): add remote list command

### LINKED ISSUE #95: remote list command

### PR DESCRIPTION
**Pull request by tareknaser360**
fixes #95

### CODE REVIEW

File: coffee_core/src/lib.rs [hunk 1]
```diff
@@ -20,6 +20,7 @@ pub enum CoffeeOperation {
 pub enum RemoteAction {
     Add(String, String),
     Rm(String),
+    List(),
```
Comment (vincenzopalazzo) [inline coffee_core/src/lib.rs:23]:
```suggestion
    List,
```

File: coffee_lib/src/plugin_manager.rs [hunk 2]
```diff
@@ -31,6 +31,9 @@ pub trait PluginManager {
     /// remove the remote repository from the plugin manager.
     async fn rm_remote(&mut self, name: &str) -> Result<(), CoffeeError>;
 
+    /// list the remote repositories for the plugin manager.
+    async fn list_remote(&mut self) -> Result<Value, CoffeeError>;
```
Comment (vincenzopalazzo) [inline coffee_lib/src/plugin_manager.rs:35]:
```suggestion
    async fn list_remotes(&mut self) -> Result<Value, CoffeeError>;
```

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
please fix also your git history!

a commit header with `docs:` but then you change also the code

**Comment by tareknaser360** [comment]
> a commit header with `docs:` but then you change also the code

Yes. I didn't notice.
Fixed

**Review by vincenzopalazzo — APPROVED** [review]

**Pull request merged**